# Patch Makefile so zlib provided by EasyBuild is picked up

--- Makefile.orig	2018-02-07 13:37:00.000000000 +0100
+++ Makefile	2018-02-07 13:37:00.000000000 +0100
@@ -1,12 +1,11 @@
-CC=gcc
-CFLAGS=-O3 -Wall -Wextra -Wno-unknown-pragmas
+ZLIB=-L$(EBROOTZLIB)/lib
 LDFLAGS=
 LIBS=-lm -lpthread -lz
 ZOPFLI=zopfli/src/zopfli/
 ZOP=deflate.o blocksplitter.o tree.o lz77.o cache.o hash.o util.o squeeze.o katajainen.o

 # use gcc and gmake on Solaris
 
 pigz: pigz.o yarn.o try.o $(ZOP)
-	$(CC) $(LDFLAGS) -o pigz pigz.o yarn.o try.o $(ZOP) $(LIBS)
+	$(CC) $(LDFLAGS) -o pigz $^ $(ZLIB) $(LIBS)
 	ln -f pigz unpigz
 
 pigz.o: pigz.c yarn.h try.h ${ZOPFLI}deflate.h ${ZOPFLI}util.h
